docs/simulation/sensitivity/configuration.md

```diff
@@ -0,0 +1,30 @@
+# Configuration
+
+## Implementation
+
```

Comment:
```suggestion
```

Reply:
Why?

Reply:
Isn't this a typo - there's no text in this section, and I don't see what we could have here in fact

Reply:
It's to put something like:
To use blabla for all sensitivity analyses, you have to configure the `sensitivity-analysis` module in your configuration file:
```yaml
sensitivity-analysis:
  default-impl-name: OpenLoadFlow
```